site/content/en/docs/Advanced/multi-cluster-allocation.md

```diff
@@ -103,6 +103,17 @@ go run examples/allocator-client/main.go --ip ${EXTERNAL_IP} \
     --multicluster true
 ```
 
+If using REST use
+
+```bash
+go run examples/allocator-client/main.go --ip ${EXTERNAL_IP} \
```

Comment:
Should this be `curl` if using REST? (Also, we'll need to wrap it in the `feature` shortcode like above please.)

Reply:
Good catch. Fixed